Marlin/language_tr.h

```diff
@@ -81,7 +83,7 @@
 #define MSG_MOVE_1MM                        _UxGT("1mm")                                                // 1mm
 #define MSG_MOVE_10MM                       _UxGT("10mm")                                               // 10mm
 #define MSG_SPEED                           _UxGT("Hız")                                                // Hız
-#define MSG_BED_Z                           _UxGT("Tabla Z")                                            // Tabla Z
+#define MSG_BED_Z                           _UxGT("Tabla-Nozül Mesafesi")                               // Tabla-Nozül Mesafesi
```

Comment:
Solved it. I think I had a problem with my version. Now Turkish characters display fine.